### PR TITLE
use prettier format ts too

### PR DESCRIPTION
**Pull request by gengjiawen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
   ],
   "scripts": {
     "clean": "rimraf lib dist es coverage",
-    "format": "prettier --write \"{src,test}/**/*.js\"",
-    "format:check": "prettier --list-different \"{src,test}/**/*.js\"",
+    "format": "prettier --write \"{src,test}/**/*.js\" \"{src,test}/**/*.ts\"",
+    "format:check": "prettier --list-different \"{src,test}/**/*.js\" \"{src,test}/**/*.ts\"",
```
Comment (eps1lon) [inline package.json]: You should be using the same pattern approach as for the folders i.e `{src,test}/**/*.{js,ts}`

Comment (gengjiawen) [reply package.json]: done.

File: package.json [hunk 2]
```diff
@@ -36,8 +36,8 @@
   ],
   "scripts": {
     "clean": "rimraf lib dist es coverage",
-    "format": "prettier --write \"{src,test}/**/*.js\"",
-    "format:check": "prettier --list-different \"{src,test}/**/*.js\"",
+    "format": "prettier --write \"{src,test}/**/*{.js,ts}\"",
+    "format:check": "prettier --list-different \"{src,test}/**/*{.js,ts}\"",
```
[the remark this answers is not in the document]
Comment (gengjiawen) [reply package.json]: Fixed.

**Comment by timdorr** [comment]
Should this also be applied to index.d.ts?

**Comment by gengjiawen** [comment]
@timdorr Fixed.

**Comment by timdorr** [comment]
Cool. Thanks!

**Pull request merged**